sites/frontend/components/Epic.js

```diff
@@ -3,7 +3,7 @@ import { styled } from '@guardian/guui';
 import pallete from '@guardian/pasteup/palette';
 import { headline } from '@guardian/pasteup/fonts';
 
-import SiteMessage from '@guardian/guui/SiteMessage';
+import SiteMessage from '@guardian/guui/components/SiteMessage';
```

Comment:
In some places we use relative module specifiers, and here we are using an absolute one. Should we try to be consistent?

Reply:
I like the idea of having absolute paths from say the `components` directory. I wanted to update the existing components in frontend to have this but found the nesting of components lead to ridiculously long paths, this was one of the reasons I'm keen to flatten the existing `<Header>` component, something I've discussed with @nicl